blog/2023-02-26-vk326.md

```diff
@@ -0,0 +1,12 @@
+---
+title: About my-cornell-netid
```

Comment:
:wave: this should be your actual net-id

Reply:
Thank you, changed it